common/json.c

```diff
@@ -187,44 +187,6 @@ const jsmntok_t *json_get_arr(const jsmntok_t tok[], size_t index)
 	return NULL;
 }
 
-/* Guide is a string with . for members, [] around indexes. */
-const jsmntok_t *json_delve(const char *buffer,
```

Comment:
I intend to eventually use this function in the pilotd, possibly.

Reply:
I vote we drop it now and remember it exists.  You can resurrect it later :)